### DISCUSSION TITLE
I have a problem with notifications

### DISCUSSION THREAD

**Discussion by Wither362** [community/community#35809] [Other Feature Feedback, Questions, & Ideas] [answered]
Ok, I commented in a discussion (that wasn’t mine), and an user replied to me, but the creator of the discussion deleted the post, so I have now an annoying ghost notification I can’t read because the discussion have been deleted. _**How do I fix this?**_.

This is the problem:

![annoying](https://user-images.githubusercontent.com/93864752/195157239-57e1bd3d-7149-4b03-acb2-016953f10136.jpeg)

**Comment by radar-blip** [comment, marked as answer]
Here is one discussion that might help:  #35381

Here is another thread with more solutions:  #6874

**Reply by Wither362** [reply]
> Here is one discussion that might help: #35381
> 
> Here is another thread with more solutions: #6874

Ye, but I don’t know how to make a curl command, how do I do it?

**Reply by Wither362** [reply]
> Here is one discussion that might help: #35381
> 
> Here is another thread with more solutions: #6874

Oh, thanks for helping me, I though I would need to sacrifice my account
